src/libs/actions/Report.js

```diff
@@ -904,9 +966,13 @@ function editReportComment(reportID, originalReportAction, textForNewComment) {
 
     // Do not autolink if someone explicitly tries to remove a link from message.
     // https://github.com/Expensify/App/issues/9090
+    // https://github.com/Expensify/App/issues/13221
+    const originalCommentHTML = lodashGet(originalReportAction, 'message[0].html');
+    const markdownForNewComment = handleUserDeletedLinks(textForNewComment, originalCommentHTML);
+
     const autolinkFilter = {filterRules: _.filter(_.pluck(parser.rules, 'name'), name => name !== 'autolink')};
-    const htmlForNewComment = parser.replace(textForNewComment, autolinkFilter);
-    const originalMessageHTML = parser.replace(originalReportAction.message[0].html, autolinkFilter);
+    const htmlForNewComment = parser.replace(markdownForNewComment, autolinkFilter);
+    const parsedOriginalCommentHTMl = parser.replace(originalCommentHTML, autolinkFilter);
```

Comment:
The last character here is still wrong, it's a lowercase `l` and should be an uppercase `L`

Reply:
oh ops sorry i just noticed it and fixed :D